### PR TITLE
Use containerapps for component-sync

### PR DESCRIPTION
**Pull request by janboll**
Add the configuration to this repo and reference it in the Dockerfile, thus add it to the image

### What this PR does

Jira: 
Link to demo recording:

### CODE REVIEW

**Comment by janboll** [comment]
Please do not merge yet

**Comment by bennerv** [comment]
@janboll converted to draft until ready for review / merge.  Feel free to convert it when ready.

**Review by geoberle — APPROVED** [review]

**Pull request merged**
